tool/clean/clean_host/clean_host.go

```diff
@@ -19,8 +16,6 @@ import (
 	"github.com/aws/amazon-cloudwatch-agent/tool/clean"
 )
 
-var REGIONS = []string{"us-east-1", "us-west-2"}
```

Comment:
Not sure why we only clean hosts in "us-east-1" and "us-west-2" this is wrong @okankoAMZ